deeptools/countReadsPerBin.py

```diff
@@ -269,6 +270,18 @@ def run(self):
             # in case a region is used, append the tilesize
             self.region += ":{}".format(self.binLength)
 
+        # Handle GTF options
+        transcriptID = "transcript"
+        exonID = "exon"
+        transcript_id_designator = "transcript_id"
```

Comment:
Can we put this block into a private function as utils, this at least two times.